doc/components/dropdown.md

```diff
@@ -26,6 +26,11 @@ like `text="Click me" icon="icon-filter"`. As following you can see an example o
   </div>
 </div>
 
+**Disabled**
+As any common html element the dropdown can be disabled by adding the `disabled="1"` flag to it.
```

Comment:
It is valid HTML but I just say it gives the wrong impression as `disabled="false"` is still disabling a button: https://codepen.io/anon/pen/owwdEp

My approach is here https://codepen.io/anon/pen/GEEdXP